packages/inngest/README.md

```diff
@@ -4,7 +4,9 @@
   <br/>
   <br/>
   <p>
-    Serverless event-driven queues, background jobs, and scheduled jobs for Typescript.<br />
+    Inngest is the reliability layer for modern applications. It combines durable execution, events, and queues into a zero-infra platform with built-in observability.
+
+    Inngest enables serverless event-driven queues, background jobs, and scheduled jobs for TypeScript.<br />
```

Comment:
Is this supposed to be here? The `<br/>` shows up in the doc.

Reply:
thank you 🫣